src/app/pages/sign-in/sign-in.module.js

```diff
@@ -3,7 +3,7 @@
 import SignInController from './sign-in.controller.js';
 import './sign-in.scss';
 
-const signInModule = angular.module('signInModule', []);
+const signInModule = angular.module('signInModule', ['toastr']);
```

Comment:
toastr - dependency of global app, not for module

Reply:
Ok, solved